test/test_queries.py

```diff
@@ -0,0 +1,68 @@
+'''
+test_queries.py
+Contains test cases for database query related functions.
+'''
+from nose.tools import assert_equal
+from components import model
+
+# HOW TO RUN NOSE TESTS
+# 1. Make sure you are in cs-modify main directory
+# 2. Make sure the path "C:\Python27\Scripts" is added in your environment variables
+# 3. Enter in cmd: "nosetests test/"
+# 4. Nose will run all the tests inside the test/ folder
+
+class TestCode(object):
+    '''
+        This class runs the test cases for database query related functions.
+    '''
+    def __init__(self):
+        self.num_column_for_each_year = 2
+        self.num_column_for_each_focus = 2
+
+    def test_query_num_students_in_year_of_study(self):
+        '''
+            Tests querying number of students at each year of study
+        '''
+        num_in_year = [4, 3, 3, 3]
+
+        table_of_year_of_study_with_count = \
+            model.get_num_students_by_yr_study()
+
+        num_of_rows_expected = 4
+        for index_row in range(1, num_of_rows_expected):
```

Comment:
I think you might have missed out the case for year 4 students or something, since <code>range(start, end)</code> excludes <code>end</code>.

Also, **it might not be safe to assume that all students will hit a maximum of <code>Year 4</code> status**. Please re-examine your implementation.